iset.mm

```diff
@@ -106184,6 +106417,73 @@ which defines the set we are taking the supremum of must (a) be true at
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                      The least common multiple
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+According to Wikipedia ("Least common multiple", 27-Aug-2020,
```

Comment:
Formatting: blank lines after =-=-=-= and before $) and indentation of 2.

Reply:
I'm adding the blank lines. As for the identation, set.mm seems to have either an identation of 0 or 2 in different places. I'm reluctant to just mess with this one and still leave things quite consistent.

Reply:
This is discussed in #2262 (but not a priority to me).